src/com/android/calendar/CalendarToolbarHandler.java

```diff
@@ -141,7 +141,14 @@ private String buildDayOfWeek() {
             dayOfWeek = DateUtils.formatDateRange(mContext, mFormatter, mMilliTime, mMilliTime,
                     DateUtils.FORMAT_SHOW_WEEKDAY, mTimeZone).toString();
         }
-        return dayOfWeek.toUpperCase();
```

Comment:
Shouldn't it be enough to remove the `.toUpperCase()` here?
Can you change your pull request accordingly?